### PR TITLE
RAC-5619: add SOL/VNC into Docker Post Test

### PR DESCRIPTION
**Pull request by panpan0000**
Background:
in CI pipeline, SOL/VNC logs were not available in post-test stage.
this PR will add the logs .

Test:
image

http://10.**.59.1**:8080/job/MasterCI_VNC/

OVA-Post-Test enhancement will come soon,

### CODE REVIEW

File: test.sh.in [hunk 1]
```diff
@@ -366,8 +366,9 @@ if [ "$RUN_FIT_TEST" == true ] ; then
     setupVirtualEnv
     waitForAPI
     nodesOn &
-    # Doesn't support ova smoke test now
-    # generateSolLog
+
+    generateSolLog
+    vnc_record_start
     # Run tests
```
Comment (PengTian0) [inline test.sh.in:373]: Please add ```trap``` to clean up the process launched by ```generateSolLog``` and ```vnc_record_start```

Comment (panpan0000) [reply test.sh.in:373]: done. thanks

**Comment by PengTian0** [comment]
:+1:

**Comment by changev** [comment]
👍

**Review by anhou — APPROVED** [review]

**Review by iceiilin — APPROVED** [review]

**Pull request merged**